### ISSUE TITLE
[Bug]: Typescript definition of method `clearBorders` of custom borders plugin asks for an argument

### ISSUE THREAD

**Issue by seleckis** [handsontable/handsontable#10515]
### Describe the bug

By [documentation](https://handsontable.com/docs/react-data-grid/api/custom-borders/#clearborders) `clearBorders` could be used without argument, but the interface of `CustomBorders` class is the following: 
```ts
export class CustomBorders extends BasePlugin {
  constructor(hotInstance: Core);
  isEnabled(): boolean;
  setBorders(selectionRanges: RangeType[][] | Array<[number, number, number, number]>, borderObject: object): void;
  getBorders(selectionRanges: RangeType[][] | Array<[number, number, number, number]>): Array<[object]>;
  clearBorders(selectionRanges: RangeType[][] | Array<[number, number, number, number]>): void;
}
```
https://github.com/handsontable/handsontable/blob/ab279a9035813c9ae585382be565f033ce0ad4eb/handsontable/types/plugins/customBorders/customBorders.d.ts#L34
seems like should be: 
```ts
...
  clearBorders(selectionRanges?: RangeType[][] | Array<[number, number, number, number]>): void;
...
```

### Video/Screenshots

<img width="895" alt="Screenshot 2023-09-19 at 14 32 34" src="https://github.com/handsontable/handsontable/assets/988504/39f8a138-80c9-4b80-ae3c-aa12335f7f53">

### Handsontable version

13.1.0

### Framework version

Next.js 13.4.12

### Your environment

MacOS Ventura 13.5.2

**Comment by adrianszymanski89** [comment]
Hi @seleckis 

Thank you for reporting this. I'll report it internally for further discussion and get back to you once we have the possible timeline for the fix.

**Comment by AMBudnik** [comment]
Hi @seleckis 

We just released Handsontable v14.1.0, where this issue is fixed.

If you would like to read more about the changes and fixed for this version please feel free to take a look at our recent blog post at https://handsontable.com/blog/handsontable-14-1-0-typescript-ssr-improvements.
